Bonjour,

Vous trouverez dans ce mail un rapport d'erreur du traitement {{ processingId }} « {{ originalName }} ».

{{ errorMessage }}

Vous pouvais retrouvé une description de la cause possible de l'erreur sur le lien ci-dessous :

https://github.com/Inist-CNRS/tdm-factory/blob/main/ERROR_CODE_FR.md

Voici un récapitulatif des informations liées à ce traitement :

- Id du traitement : {{ processingId }}
- Nom du fichier d'origine : {{ originalName }}
- Convertisseur : {{ wrapper }}
- Paramètre du convertisseur : {{ wrapperParam }}
- Enrichissement : {{ enrichment }}


Service IA Factory,
Merci de ne pas répondre.
